crates/hotshot-qc/src/bit_vector_old.rs

```diff
@@ -263,12 +263,12 @@ mod tests {
             // Check the QC and the QCParams can be serialized / deserialized
             assert_eq!(
                 qc,
-                bincode::deserialize(&bincode::serialize(&qc).unwrap()).unwrap()
+                Serializer::<0, 1>::deserialize(&Serializer::<0, 1>::serialize(&qc).unwrap()).unwrap()
```

Comment:
Since this a round-trip tests that should really work in any version, should this (and L271 in the same file) be `Serializer::CURRENT_STATIC_VERSION`? (Should we have a `CURRENT_STATIC_VERSION`? We could alias it to `STATIC_VERSION_0_1`, but then we wouldn't have to update it here.)

Reply:
There does need to be a `CURRENT_BASE_VERSION`, and a `CURRENT_UPGRADE_VERSION`, and a compile-time check to determine if they are the same (e.g. no upgrade available), but aliasing won't work. I'm working on a solution using a generic type to impl on the core types, but I haven't worked it out yet. I left this (and a few other places) as-is because they're not actually hooked up to the real versioning.